src/test/components/layout/header/tag-mapper.test.ts

```
@@ -0,0 +1,305 @@
+import { describe, it, expect, vi, beforeEach, afterEach } from "vitest";
+import { TagMapper } from "@/components/layout/header/page-mappers/tag-mapper";
+import type { TagPageInfo, PageInfo } from "@/components/layout/header/page-detectors/types";
+import type { TagTranslations } from "@/lib/article/tag-utils";
```

ESLint

[error] 'TagTranslations' is defined but never used.

(@typescript-eslint/no-unused-vars)